client/pom.xml

```diff
@@ -177,6 +177,11 @@
             <artifactId>snakeyaml</artifactId>
             <version>1.32</version>
         </dependency>
+        <dependency>
```

Comment:
why we need a new dependency for parsing JSON?

Reply:
We need that one to map the empty elements or elements that aren't.